src/tabs/RobotSimulation/components/Simulation/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Tabs } from '@blueprintjs/core';
+import { useRef, type CSSProperties, useEffect, useState } from 'react';
+
+import { type World } from '../../../../bundles/robot_simulation/engine';
+import { type WorldState } from '../../../../bundles/robot_simulation/engine/World';
+import type { DebuggerContext } from '../../../../typings/type_helpers';
+
+const WrapperStyle: CSSProperties = {
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.6rem',
+};
+
+const CanvasStyle: CSSProperties = {
+  width: 900,
+  height: 500,
+  borderRadius: 3,
+  overflow: 'hidden',
+  boxShadow: 'inset 0 0 0 1px rgba(255, 255, 255, 0.2)',
+};
+
+const bottomPanelStyle: CSSProperties = {
+  width: 900,
+  height: 200,
+  backgroundColor: '#1a2530',
+  borderRadius: 3,
+  overflow: 'hidden',
+  boxShadow: 'inset 0 0 0 1px rgba(255, 255, 255, 0.2)',
+};
+
+export default function SimulationCanvas({
+  context,
+  isOpen,
+}: {
+  context: DebuggerContext;
+  isOpen: boolean;
+}) {

```

Comment:
This is a component as well right? Can we use React.FC typing?

Reply:
Fixed!